rasa/telemetry.py

```diff
@@ -566,6 +566,44 @@ def toggle_telemetry_reporting(is_enabled: bool) -> None:
     rasa_utils.write_global_config_value(CONFIG_FILE_TELEMETRY_KEY, configuration)
 
 
+def filter_errors(
+    event: Dict[Text, Any], hint: Optional[Dict[Text, Any]] = None
+) -> Optional[Dict[Text, Any]]:
+    """Filter errors.
+
+    Args:
+        event: event to be logged to sentry
+        hint: some hinting information sent alongside of the event
+
+    Returns:
+        the event without any sensitive / PII data or `None` if the event constitutes
+        an `ImportError` which should be discarded.
+    """
+    if "exc_info" in hint:
```

Comment:
@ancalita re the code quality error
```suggestion
    if hint and "exc_info" in hint:
```